src/shared/ui/list/list.tsx

```diff
@@ -0,0 +1,44 @@
+// todo delete TextWithLink
```

Comment:
There was a comment above TODO, just in case I'll highlight it too

Reply:
I agree with you, but in this case such a problem will not arise, since we have to refactor all `*.data` files and there is no point in creating another additional task for this changes